README.rdoc

```diff
@@ -17,6 +17,49 @@ Include ElasticRecord into your model:
     include ElasticRecord::Model
   end
 
+== Gem Configuration
+
+There are global configuration options for ElasticRecord that should be initialized before any
+models are declared.
+
+=== Connection Options (Required)
+
+There are two ways to set up which server(s) to connect to:
+
+* servers, via code:
+    ElasticRecord.configure do |config|
+      config.servers = %w(es1.example.com:9200 es2.example.com:9200)
+    end
+
+* connection_options, either via code or a YAML file at config/elasticsearch.yml if you are using Rails.
+  For example, using the YAML format:
+
+      development:
+        servers: es1.example.com:9200,es2.example.com:9200
+        timeout: 10
+        retries: 2
+
+=== Index Management
+
+If you need to manage multiple indexes via the rake tasks, you will need to declare them explicitly:
+
+  ElasticRecord.configure do |config|
+    config.model_names = %w(Product Order Location)
+  end
+
+=== JSON Adapter
+
+By default, ElasticRecord uses ActiveSupport::JSON to (de)serialize Elasticsearch payloads. There
+is also optional support for using the Oj gem. To use Oj, ensure that oj is required and set:
+
+  ElasticRecord::JSON.parser = :oj
+
+To return to the default parser, set the variable to :active_support.
+
+=== Other Settings
+
+* scroll_keep_alive - the amount of time a scroll exists
```

Comment:
The docs are unclear about where this goes. Either delete or include in the connection options section.